lib/gmock-1.7.0/gtest/test/gtest-test-part_test (1).cc

```diff
@@ -0,0 +1,208 @@
+// Copyright 2008 Google Inc.
```

Comment:
Hello Be,

As I said, I had a real hard time with Github... Actually, I find this to be the hardest thing to do in contributing. I tried to do a clean fork for this, but it keep bringing me to the old fork I had. Can I simply send you the .js file so that you can integrate it?
Thanks